sdk/education/azure-mgmt-education/azure/mgmt/education/aio/operations/_create_operations.py

```diff
@@ -0,0 +1,196 @@
+
+    :ivar models: Alias to model classes used in this operation group.
+    :type models: ~azure.mgmt.education.models
+    :param client: Client for service requests.
+    :param config: Configuration of service client.
+    :param serializer: An object model serializer.
+    :param deserializer: An object model deserializer.
+    """
+
+    models = _models
+
+    def __init__(self, client, config, serializer, deserializer) -> None:
+        self._client = client
+        self._serialize = serializer
+        self._deserialize = deserializer
+        self._config = config
+
+    @distributed_trace_async
+    async def lab(
+        self,
+        billing_account_name: str,
+        billing_profile_name: str,
+        invoice_section_name: str,
+        parameters: "_models.LabDetails",
+        **kwargs: Any
+    ) -> "_models.LabDetails":
+        """Create a new lab or update a previously created lab.
+
+        :param billing_account_name: Billing account name.
+        :type billing_account_name: str
+        :param billing_profile_name: Billing profile name.
+        :type billing_profile_name: str
+        :param invoice_section_name: Invoice section name.
+        :type invoice_section_name: str
+        :param parameters: Request parameters that are provided to create lab resource.
+        :type parameters: ~azure.mgmt.education.models.LabDetails
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: LabDetails, or the result of cls(response)
+        :rtype: ~azure.mgmt.education.models.LabDetails
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        cls = kwargs.pop('cls', None)  # type: ClsType["_models.LabDetails"]
+        error_map = {
+            401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError
+        }
+        error_map.update(kwargs.pop('error_map', {}))
+
+        api_version = kwargs.pop('api_version', "2021-12-01-preview")  # type: str
+        content_type = kwargs.pop('content_type', "application/json")  # type: Optional[str]
+
+        _json = self._serialize.body(parameters, 'LabDetails')
+
+        request = build_lab_request(
+            billing_account_name=billing_account_name,
+            billing_profile_name=billing_profile_name,
+            invoice_section_name=invoice_section_name,
+            api_version=api_version,
+            content_type=content_type,
+            json=_json,
+            template_url=self.lab.metadata['url'],
+        )
+        request = _convert_request(request)
+        request.url = self._client.format_url(request.url)
+
+        pipeline_response = await self._client._pipeline.run(  # pylint: disable=protected-access
+            request,
+            stream=False,
+            **kwargs
+        )
+        response = pipeline_response.http_response
+
+        if response.status_code not in [200, 201]:
```

Comment:
eduRP create and update endpoint is going to be returning 201 and 202, this will be failing is 202 is returned.
How do we fix this?

Reply:
Hi Yuheng, Python SDK is generated based on the [rest API](https://github.com/Azure/azure-rest-api-specs), so to add 202 response, we need to define it [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/education/resource-manager/Microsoft.Education/preview/2021-12-01-preview/education.json#L410) first, and then we regenerate it.